clicommand/artifact_shasum.go

```diff
@@ -68,6 +74,10 @@ var ArtifactShasumCommand = cli.Command{
 	Usage:       "Prints the SHA-1 hash for a single artifact specified by a search query",
 	Description: ShasumHelpDescription,
 	Flags: []cli.Flag{
+		cli.BoolFlag{
+			Name:  "sha256",
+			Usage: "Request SHA-256 instead of SHA-1, errors if SHA-256 not available",
+		},
```

Comment:
is it worth preferentially fetching SHA256, and then returning SHA1 only if it isn't present, rather than making users decide? would that break any workflows?

Reply:
It might break things, if e.g. people were comparing it against their own SHA-1.